### ISSUE TITLE
Config.unobserve is deprecated.

### ISSUE THREAD

**Issue by snkisuke** [AtomLinter/linter-pylint#31]
Config::unobserve no longer does anything. Call `.dispose()` on the object returned by Config::observe instead.
```
Config.unobserve (/opt/atom/resources/app/src/config.js:1045:19)
LinterPylint.destroy (/home/snkisuke/.atom/packages/linter-pylint/lib/linter-pylint.coffee:32:16)
```

**Comment by karimjosephjr** [comment]
Also in atom -v 0.198.0, linter-pylint 0.2.1

**Comment by langerma** [comment]
also in atom 0.200.0

**Comment by kevinsawicki** [comment]
Fixed by https://github.com/AtomLinter/linter-pylint/pull/40 and published in `0.2.2`
